<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAdd</class>
 <widget class="QDialog" name="DialogAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="typeTriangle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>triangle</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dlbCenterX">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dlbCenterY">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dblRadius">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>65</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CenterX</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CenterY</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Radius</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="typeRectangle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>rectangle</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="typeCircle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>circle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAdd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAdd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
